Shortcut creation failure

### Expected behavior
Creating a shortcut to an existing prefix. Creating shortcut during script installation works fine.
### Actual behavior
Upon clicking "save" I got this error:
```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.inject.internal.cglib.core.$ReflectUtils$1 (file:/usr/share/maven/lib/guice.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of com.google.inject.internal.cglib.core.$ReflectUtils$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[INFO] Scanning for projects...
[INFO] 
[INFO] -------------------< org.phoenicis:phoenicis-javafx >-------------------
[INFO] Building phoenicis-javafx 5.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- exec-maven-plugin:1.6.0:java (default-cli) @ phoenicis-javafx ---
ERROR StatusLogger No Log4j 2 configuration file found. Using default configuration (logging only errors to the console), or user programmatically provided configurations. Set system property 'log4j2.debug' to show Log4j 2 internal initialization logging. See https://logging.apache.org/log4j/2.x/manual/configuration.html for instructions on how to configure Log4j 2
[INFO ] org.phoenicis.repository.DefaultRepositoryManager (l.133) - Adding repositories: [org.phoenicis.repository.location.LocalRepositoryLocation@7bd0ab5d[/home/jonasz/Pobrane/scripts-master]] at index 0
[INFO ] org.phoenicis.repository.types.MultipleRepository (l.46) - Fetching applications for: org.phoenicis.repository.types.MultipleRepository@492e97dc[repositories=[org.phoenicis.repository.types.LocalRepository@23255101[repositorySource=file:/home/jonasz/Pobrane/scripts-master/,repositoryDirectory=/home/jonasz/Pobrane/scripts-master]]]
[INFO ] org.phoenicis.repository.types.LocalRepository (l.77) - Reading directory : /home/jonasz/Pobrane/scripts-master
[ERROR] org.phoenicis.multithreading.ControlledThreadPoolExecutorService (l.64) - java.lang.ClassCastException: class java.lang.String cannot be cast to class org.graalvm.polyglot.Value (java.lang.String is in module java.base of loader 'bootstrap'; org.graalvm.polyglot.Value is in unnamed module of loader java.net.URLClassLoader @76c548f)
	at org.phoenicis.javafx.components.library.control.LibraryFeaturePanel.lambda$createShortcut$0(LibraryFeaturePanel.java:157)
	at org.phoenicis.scripts.session.PhoenicisInteractiveScriptSession.eval(PhoenicisInteractiveScriptSession.java:35)
	at org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$createInteractiveSession$1(BackgroundScriptInterpreter.java:45)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

[WARNING] 
java.lang.ClassCastException: class java.lang.String cannot be cast to class org.graalvm.polyglot.Value (java.lang.String is in module java.base of loader 'bootstrap'; org.graalvm.polyglot.Value is in unnamed module of loader java.net.URLClassLoader @76c548f)
    at org.phoenicis.javafx.components.library.control.LibraryFeaturePanel.lambda$createShortcut$0 (LibraryFeaturePanel.java:157)
    at org.phoenicis.scripts.session.PhoenicisInteractiveScriptSession.eval (PhoenicisInteractiveScriptSession.java:35)
    at org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$createInteractiveSession$1 (BackgroundScriptInterpreter.java:45)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1128)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:628)
    at java.lang.Thread.run (Thread.java:834)
```
Terminal log:
```
(java:6266): Gdk-WARNING **: 09:58:29.941: XSetErrorHandler() called with a GDK error trap pushed. Don't do that.
[ERROR] org.phoenicis.multithreading.ControlledThreadPoolExecutorService (l.64) - java.lang.ClassCastException: class java.lang.String cannot be cast to class org.graalvm.polyglot.Value (java.lang.String is in module java.base of loader 'bootstrap'; org.graalvm.polyglot.Value is in unnamed module of loader java.net.URLClassLoader @76c548f)
	at org.phoenicis.javafx.components.library.control.LibraryFeaturePanel.lambda$createShortcut$0(LibraryFeaturePanel.java:157)
	at org.phoenicis.scripts.session.PhoenicisInteractiveScriptSession.eval(PhoenicisInteractiveScriptSession.java:35)
	at org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$createInteractiveSession$1(BackgroundScriptInterpreter.java:45)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

[WARNING] 
java.lang.ClassCastException: class java.lang.String cannot be cast to class org.graalvm.polyglot.Value (java.lang.String is in module java.base of loader 'bootstrap'; org.graalvm.polyglot.Value is in unnamed module of loader java.net.URLClassLoader @76c548f)
    at org.phoenicis.javafx.components.library.control.LibraryFeaturePanel.lambda$createShortcut$0 (LibraryFeaturePanel.java:157)
    at org.phoenicis.scripts.session.PhoenicisInteractiveScriptSession.eval (PhoenicisInteractiveScriptSession.java:35)
    at org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$createInteractiveSession$1 (BackgroundScriptInterpreter.java:45)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1128)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:628)
    at java.lang.Thread.run (Thread.java:834)
```
### Steps to reproduce
Click on "create a shortcut" button and fill the necesseary fields. Click on save.
### System information
* software version (git revision) of phoenicis and scripts repositories
https://github.com/PhoenicisOrg/phoenicis/commit/f2c02539a40d8b985aca99d3bfec0c584f097974
https://github.com/PhoenicisOrg/scripts/commit/3a260960d1b424631c5b3c85bd3cb782204f144e
* used operating system
Ubuntu 19.10
* used Java version
```
openjdk 11.0.5 2019-10-15
OpenJDK Runtime Environment (build 11.0.5+10-post-Ubuntu-0ubuntu1.1)
OpenJDK 64-Bit Server VM (build 11.0.5+10-post-Ubuntu-0ubuntu1.1, mixed mode, sharing)
```

--- a/phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/library/control/LibraryFeaturePanel.java
+++ b/phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/library/control/LibraryFeaturePanel.java
@@ -152,7 +152,7 @@ public class LibraryFeaturePanel extends FeaturePanel<LibraryFeaturePanel, Libra
                 + "\");";
 
         interactiveScriptSession.eval(scriptInclude,
-                ignored -> interactiveScriptSession.eval("new " + engine + "Shortcut()",
+                ignored -> interactiveScriptSession.eval("new Shortcut()",
                         output -> {
                             final Value shortcutObject = (Value) output;
 

